### DISCUSSION TITLE
apache server works locally but not from outside

### DISCUSSION THREAD

**Discussion by Ostreey** [osm-search/Nominatim#2783] [General]
ubuntu VERSION="20.04.4 LTS server Aruba Cloud

Hi, i've succesfully imported Nominatim database. Development server works, apache2 server works but only locally, meaning : 
`curl` `'localhost/nominatim/search.php?q=Manta'`

gives me reply with data

`curl` ``'my-public-ip/nominatim/search.php?q=Manta'`

gives me

404 Not Found
Not Found
The requested URL was not found on this server.

Apache/2.4.41 (Ubuntu) Server at 80.211.254.59 Port 80
My /etc/apache2/conf-available/nominatim.conf file is:

`<Directory "srv/nominatim/website">`
`  Options FollowSymLinks MultiViews`
 ` AddType text/html   .php`
  `DirectoryIndex search.php`
 ` AllowOverride All`
  `Require all granted`
`</Directory>`

`Alias /nominatim  srv/nominatim/website`

My /etc/apache2/apache2.conf is:

`<Directory />`
 `       Options FollowSymLinks`
  `      AllowOverride All`
   `     Require all granted`
`</Directory>`

`<Directory /usr/share>`
 `       AllowOverride All`
  `      Require all granted`
`</Directory>`

`<Directory /var/www/>`
 `       Options Indexes FollowSymLinks`
  `      AllowOverride All`
   `     Require all granted`
``</Directory>`
after changes i did:

`sudo a2dismod rewrite`
and

`sudo systemctl restart apache2`
Last lines from apache access.log:

`83.0.66.104 - - [29/Jul/2022:12:41:10 +0000] "GET /nominatim/search.php?q=Manta HTTP/1.1" 404 492 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
83.0.66.104 - - [29/Jul/2022:12:42:02 +0000] "-" 408 0 "-" "-"
83.0.66.104 - - [29/Jul/2022:12:42:03 +0000] "GET /nominatim/search.php?q=Quito HTTP/1.1" 404 492 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"'
ss -lntp result:

'State                Recv-Q               Send-Q                             Local Address:Port                             Peer Address:Port              Process
LISTEN               0                    4096                               127.0.0.53%lo:53                                    0.0.0.0:*                  users:(("systemd-resolve",pid=710,fd=13))
LISTEN               0                    128                                      0.0.0.0:22                                    0.0.0.0:*                  users:(("sshd",pid=812,fd=3))
LISTEN               0                    511                                      0.0.0.0:1880                                  0.0.0.0:*                  users:(("node",pid=734,fd=19))
LISTEN               0                    244                                    127.0.0.1:5432                                  0.0.0.0:*                  users:(("postgres",pid=878,fd=3))
LISTEN               0                    128                                         [::]:22                                       [::]:*                  users:(("sshd",pid=812,fd=4))
LISTEN               0                    511                                            *:80                                          *:*                  users:(("apache2",pid=1758,fd=4),("apache2",pid=1747,fd=4),("apache2",pid=1746,fd=4),("apache2",pid=1745,fd=4),("apache2",pid=1744,fd=4),("apache2",pid=1743,fd=4),("apache2",pid=1742,fd=4))`

I'm new to linux. Any help appreciated. Thanks

**Comment by mtmail** [comment]
> `Alias /nominatim srv/nominatim/website`

This line looks wrong. Is the directory in `/srv/nominatim/website` instead?
